crates/metaslang/cst/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "metaslang_cst"
```

Comment:
Are we planning on introducing other `metaslang_XXX` crates?
Not sure I understand the reason behind the specialization here.

Reply:
Yes, `metaslang_graph_builder` is right after this one.

Reply:
And they don't belong in `codegen`

Reply:
Makes sense. Thanks.